analysis/common_variables.py

```diff
@@ -636,6 +638,204 @@ def generate_common_variables(index_date_variable, exposure_end_date_variable, o
         "tmp_out_date_migraine_snomed", "tmp_out_date_migraine_hes", "tmp_out_date_migraine_death",
     ),
 
+#######################################
+# Neurodegenerative History variables # 
+#######################################
+
+## History of any dementia
+    # Dementia
+    # History of Alzheimer's disease
+    # Primary
+    tmp_out_date_history_alzheimer_disease_snomed=patients.with_these_clinical_events(
```

Comment:
These arnt creating outcome or date variables - how about tmp_cov_bin_history....... ? They'll be used to exclude people but also as covariates in other analysis

Reply:
Changed prefix.